src/skin/skinloader.cpp

```diff
@@ -56,12 +56,25 @@ QString SkinLoader::getSkinPath(const QString& skinName) const {
     return QString();
 }
 
-QPixmap SkinLoader::getSkinPreview(const QString& skinName) const {
-    QPixmap preview(getSkinPath(skinName) + "/preferences_preview_screenshot.png");
+QPixmap SkinLoader::getSkinPreview(const QString& skinName, const QString& schemeName) const {
+		qDebug() << "SkinLoader::getSkinPreview  skinName =" << skinName << "schemeName =" << schemeName;
+		QPixmap preview(":/images/skin_preview_placeholder.png");
+    if (!schemeName.isEmpty()) {
+    	QString schemeNameUnformatted = schemeName;
+    	qDebug() << "SkinLoader::getSkinPreview  schemeNameUnformatted =" << schemeNameUnformatted;
+    	QString schemeNameFormatted = schemeNameUnformatted.replace(" ","");
+    	qDebug() << "SkinLoader::getSkinPreview  schemeNameFormatted =" << schemeNameFormatted;
+    	QPixmap preview(getSkinPath(skinName)+"/skin_preview_"+schemeNameFormatted+".png");
+    	qDebug() << "Looking for: "<<getSkinPath(skinName)<<"/skin_preview_"<<schemeNameFormatted<<".png";
+
+    } else {
+//    	QPixmap preview(getSkinPath(skinName) + "/skin_preview.png");
```

Comment:
`Debug [Main]: Looking for: "[install path]/share/mixxx/skins/Deere" /skin_preview.png`
file is there but it still shows the preview placeholder :\